brevbaker/src/test/kotlin/no/nav/pensjon/brev/template/render/Fixtures.kt

```diff
@@ -1,40 +1,35 @@
 package no.nav.pensjon.brev.template.render
 
+import no.nav.brev.brevbaker.BrukerTestImpl
+import no.nav.brev.brevbaker.FellesTestImpl
+import no.nav.brev.brevbaker.FoedselsnummerTestImpl
+import no.nav.brev.brevbaker.NavEnhetTestImpl
+import no.nav.brev.brevbaker.SignerendeSaksbehandlereTestImpl
+import no.nav.brev.brevbaker.TelefonnummerTestImpl
 import no.nav.pensjon.brev.template.HasModel
-import no.nav.pensjon.brev.template.LangBokmal
 import no.nav.pensjon.brev.template.Language
-import no.nav.pensjon.brev.template.Language.Bokmal
-import no.nav.pensjon.brev.template.dsl.OutlineOnlyScope
-import no.nav.pensjon.brev.template.dsl.createTemplate
 import no.nav.pensjon.brev.template.dsl.helpers.TemplateModelHelpers
-import no.nav.pensjon.brev.template.dsl.languages
 import no.nav.pensjon.brev.template.dsl.newText
-import no.nav.pensjon.brevbaker.api.model.Bruker
-import no.nav.pensjon.brevbaker.api.model.Felles
-import no.nav.pensjon.brevbaker.api.model.Foedselsnummer
 import no.nav.pensjon.brevbaker.api.model.LetterMetadata
-import no.nav.pensjon.brevbaker.api.model.NAVEnhet
-import no.nav.pensjon.brevbaker.api.model.SignerendeSaksbehandlere
-import no.nav.pensjon.brevbaker.api.model.Telefonnummer
 import java.time.LocalDate
 
 object Fixtures {
 
-    val felles = Felles(
+    val felles = FellesTestImpl(
```

Comment:
Hvorfor har vi en egen test implementasjon?

Reply:
Tanken var at vi da tilbyr denne til konsumentane for å gjera det enkelt for dei å lage testar. Men usikker på om den tanken står seg så godt, og om det er betre at dei heller bruker sine eigne implementasjonar også i testane sine.